multicluster/cmd/multicluster-controller/leader.go

```diff
@@ -114,15 +113,14 @@ func runLeader(o *Options) error {
 		return fmt.Errorf("error creating ResourceExport webhook: %v", err)
 	}
 
-	staleController := multiclustercontrollers.NewStaleResCleanupController(
+	staleController := leader.NewStaleResCleanupController(
 		mgr.GetClient(),
 		mgr.GetScheme(),
-		env.GetPodNamespace(),
-		nil,
-		multiclustercontrollers.LeaderCluster,
 	)
-
-	go staleController.Run(stopCh)
+	if err = staleController.SetupWithManager(mgr); err != nil {
+		return fmt.Errorf("error creating StaleResCleanupController: %v", err)
+	}
+	go staleController.RunPeriodically(stopCh)
```

Comment:
out of curiosity, `Run` is a common name for such long running routines, why adding a suffix for this case alone?

Reply:
Let me change it back, there was another function named RunOnce(), so I changed this name before.

Reply:
Renamed